pipeline/__init__.py

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
Why was it necessary to create this file?

Reply:
`__init__.py `lets python know that it's working with a package.